### PR TITLE
Edit profile page

### PR DESCRIPTION
**Pull request by fares98**
Create an edit profile page

### CODE REVIEW

**Comment by Fatmasiam** [comment]
fix the style and the accessblity 

Screenshot from 2019-11-05 09-13-55

File: src/containers/EditAccount/editAccount.jsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    const { checked } = this.state;
+    this.setState({ checked: !checked });
+  };
+
+  onChange = e => {
+    this.setState({
+      checked: e.target.checked,
+    });
+  };
+
```
Comment (Fatmasiam) [inline src/containers/EditAccount/editAccount.jsx]: this code must be on the js file not on the jsx

File: src/containers/EditAccount/editAccount.jsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  render() {
+    const { checked } = this.state;
+    const handleSubmit = e => {
+      e.preventDefault();
+      // submit the data
+    };
```
Comment (Fatmasiam) [inline src/containers/EditAccount/editAccount.jsx]: why this function here ?!!!

File: src/containers/EditAccount/editAccount.jsx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+  toggleChecked = () => {
+    const { checked } = this.state;
+    this.setState({ checked: !checked });
+  };
+
+  onChange = e => {
+    this.setState({
+      checked: e.target.checked,
+    });
+  };
```
Comment (Fatmasiam) [inline src/containers/EditAccount/editAccount.jsx]: try to use es6
```js
 onChange = ({ target : { checked }}) =>  this.setState({ checked: checked });
  
```

File: src/containers/EditAccount/editAccount.jsx [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+          <section className="edit-account__checkbox">
+            <Checkbox checked={checked} onChange={handleChange}>
+              Change Password
+            </Checkbox>
+          </section>
+
+          <Form.Item
+            hasFeedback
+            className={checked ? 'transition' : 'transition hidden'}
```
Comment (yosefanajjar) [inline src/containers/EditAccount/editAccount.jsx]: here the password input is going to have valued the whole time which is wrong, we don't want to have an empty value for the password!

**Review by yosefanajjar — APPROVED** [review]

**Review by alaa-yasin — APPROVED** [review]

**Review by AlaaTaima — APPROVED** [review]

**Review by Fatmasiam — APPROVED** [review]

**Pull request merged**